Public/Sdk/Public/Managed/Tools/ResGen.Lite/resgen.lite.dsc

```diff
@@ -23,17 +23,17 @@ namespace Tool {
                 importFrom("System.Threading.Tasks.Extensions").pkg,
 
                 ] : []),
-            importFrom("Microsoft.CodeAnalysis.Common").pkg,
-            importFrom("Microsoft.CodeAnalysis.CSharp").pkg,
-            importFrom("Microsoft.CodeAnalysis.CSharp.Workspaces").pkg,
-            importFrom("Microsoft.CodeAnalysis.Workspaces.Common").pkg,
+            importFrom("Microsoft.CodeAnalysis.Common").withQualifier({ targetFramework: "netstandard2.0" }).pkg,

```

Comment:
>ier({ targetFramework [](start = 66, length = 21)

Just curious why we are forcing netstandard here rather than let nuget decide the 'best' one...
Would you mind adding a little comment why?

Reply:
With the change (as described in the PR introduction) we deliberately split full framework and `.NETCore` / `.NETStandard` assemblies in the spec generation. There is no more "we build for 472, so choose `.NETStandard` 1.0 - 2.0 assemblies". The CodeAnalysis packages all come with .NETStandard assemblies only, hence we have to force the qualifier here to use the .NETStandard assemblies from the package. I think it is cleaner this way as it communicates intent by the author of the DScript file and indicates what assemblies to use. Anther consideration is we are switching to .NETCoreApp only soon, at that point the qualifier enforcement can be removed all together. I'll leave a comment too.